config/ember-try.js

```diff
@@ -1,134 +1,94 @@
-/*jshint node:true*/
-module.exports = {
-  useYarn: true,
-  scenarios: [
-    {
-      name: 'default',
-      bower: {
-        dependencies: {},
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+
+module.exports = async function() {
+  return {
+    useYarn: true,
+    scenarios: [
+      {
+        name: 'ember-lts-3.12',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.12.0',
+            'ember-data': '~3.12.0'
```

Comment:
@igorT sufficient? (see below for others)

Reply:
Ping @igorT

Reply:
@igorT biweekly ping ;)

Reply:
👀